os/board/rtl8721csm/src/rtl8721csm_boot.c

```diff
@@ -236,6 +236,11 @@ void board_initialize(void)
 		wifi_config.wifi_app_ctrl_tdma == FALSE) {
 		SystemSetCpuClk(CLK_KM4_100M);
 	}
+
+#ifdef CONFIG_SYSTEM_REBOOT_REASON
+	up_reboot_reason_init();
```

Comment:
Please include the `reboot_reason.h` which is in os/arch/arm/include.

Reply:
Is it <arch/reboot_reason.h> or <tinyara/reboot_reason.h> ? 
Thanks

Reply:
<arch/reboot_reason.h>

Reply:
I mean in arch. Because the prototype of `up_reboot_reason_init()` is in arch.

Reply:
OK, thank you all.